src/components/App/App.css

```diff
@@ -1,38 +1,49 @@
+/* MOBILE-FIRST VIEW */
 .App {
   text-align: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+header {
+  font-family: 'MedievalSharp', cursive;
+  font-size: x-large;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  background-color: #131316;
+  padding-bottom: 20px;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.home-button {
+  text-decoration: none;
+  color: #EF81E5;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+.home-button:hover {
```

Comment:
I really like this home button hover effect! Definitely think we should keep it.

I almost like it so much, I think it would be neat to somehow implement on the nav bar buttons for it to be neon highlighted when the user is on that current page, but might be toooo much.

Reply:
@Sykogst I'm glad you like it! I certainly could add something like it to the buttons. I just wasn't sure if that would be too much or not. And I was kinda thinking of it like the app title was a "neon sign" in a way. Maybe this is one we should discuss as a group and then I can record whatever choice we make here in the PR along with changes (if any).

Reply:
I like the neon sign idea, we can play with that as we go along cuz I think especially as a hover state it's effective for navigation clarity.

Reply:
Personally, I think it's ideal for one thing to stand out as special. But we can revisit using some level of it elsewhere, too. Maybe we can hold off on adding more of it until our final styling ticket.